easy-ui-react/src/NexusLayout/NexusLayoutMultipage.module.scss

```diff
@@ -15,6 +15,6 @@
 }
 
 .content {
-  flex: 1 0 auto;
+  flex: 1;
```

Comment:
tbh, not entirely sure if this is the appropriate solution; will likely need to raise it to steve when hes back, but for now it keeps the grow property in place and doesn't cause an overflow